synnax/pkg/access/rbac/enforce_test.go

```diff
@@ -44,33 +45,43 @@ var _ = Describe("enforcer", func() {
 		var (
 			userObject, rbacObject = user.OntologyID(uuid.New()), rbac.OntologyID(uuid.New())
 			userTypeObject         = ontology.ID{Type: "user", Key: ""}
-			user1, user2, user3    = rbac.OntologyID(uuid.New()), rbac.OntologyID(uuid.New()), rbac.OntologyID(uuid.New())
+			rbac1, rbac2, rbac3    = rbac.OntologyID(uuid.New()), rbac.OntologyID(uuid.New()), rbac.OntologyID(uuid.New())
 		)
 		BeforeEach(func() {
 			policies := []rbac.Policy{
 				{
-					Subjects: []ontology.ID{user1},
+					Subjects: []ontology.ID{rbac1},
```

Comment:
These are just for testing the enforcement mechanisms. Currently, a subject can be any ontology ID. Do you think it should be changed so that the subject is a key / username of a user instead?

Reply:
Ok, as long as we're not using this actual policy enforcement I'm ok with it.